infrastructure/terraform/components/cdn/lambda_permission_cloudfront_lambda_remove_origin_request_path.tf

```diff
@@ -0,0 +1,7 @@
+# resource "aws_lambda_permission" "cloudfront_lambda_remove_origin_request_path" {
```

Comment:
do we want to remove this?

Reply:
Yea, unlike API Gateway, CF creates the trigger automatically. ive dropped the file. Good spot